File: CS/HideableGroupRowFooters/CustomGridControl/MyGridView.cs
using DevExpress.XtraGrid;
using DevExpress.XtraGrid.Views.Grid;

namespace HideableGroupRowFooters
{
	public class MyGridView : GridView
	{
		public MyGridView(GridControl ownerGrid)
			: base(ownerGrid)
		{
		}
		public MyGridView()
			: base()
		{
		}

		internal void RaiseViewInfoShowGroupFooter(object sender, ShowGroupFooterEventArgs e)
		{
			if ( ShowGroupFooter != null )
				ShowGroupFooter(sender, e);
		}

		protected override string ViewName
		{
			get
			{
				return "MyGridView";
			}
		}

		public event ShowGroupFooterEventHandler ShowGroupFooter;
	}
}

File: VB/HideableGroupRowFooters/CustomGridControl/MyGridView.vb
Imports DevExpress.XtraGrid
Imports DevExpress.XtraGrid.Views.Grid

Namespace HideableGroupRowFooters

    Public Class MyGridView
        Inherits GridView

        Public Sub New(ByVal ownerGrid As GridControl)
            MyBase.New(ownerGrid)
        End Sub

        Public Sub New()
            MyBase.New()
        End Sub

        Friend Sub RaiseViewInfoShowGroupFooter(ByVal sender As Object, ByVal e As ShowGroupFooterEventArgs)
            RaiseEvent ShowGroupFooter(sender, e)
        End Sub

        Protected Overrides ReadOnly Property ViewName As String
            Get
                Return "MyGridView"
            End Get
        End Property

        Public Event ShowGroupFooter As ShowGroupFooterEventHandler
    End Class
End Namespace
